src/tikzdocument.jl

```diff
@@ -166,19 +178,32 @@ end
 global _tikzid = round(UInt64, time() * 1e6)
 
 if HAVE_PDFTOSVG
-    function savesvg(filename::String, td::TikzDocument; latex_engine = latexengine(),
-                                                         buildflags = vcat(DEFAULT_FLAGS, CUSTOM_FLAGS))
-        tmp = tempname()
-        keep_pdf = isfile(filename * ".pdf")
```

Comment:
I don't remember what the point of this `keep_pdf` was. It seems useless. Should probably be completely removed.

Reply:
Perhaps in a later PR, I want to focus on writing tons of tests first.